.github/workflows/e2e-test-weekly.yml

```diff
@@ -260,6 +283,7 @@ jobs:
           selfManagedInfra: ${{ matrix.selfManagedInfra == 'true' }}
           s3AccessKey: ${{ secrets.AWS_ACCESS_KEY_ID_S3PROXY }}
           s3SecretKey: ${{ secrets.AWS_SECRET_ACCESS_KEY_S3PROXY }}
+          internalLoadBalancer: ${{ matrix.internal-lb == 'true' }}
```

Comment:
```suggestion
          internalLoadBalancer: ${{ matrix.internal-lb }}
```

This doesn't work?

Reply:
I just copied what we do for `matrix.selfManagedInfra`. Hence the variable is a string and only truthy. But I don't know if this is equal to your suggestion.